mkt/constants/features.py

```diff
@@ -335,6 +336,12 @@
                              u'1GB RAM.'),
         'apis': (),
     }),
+    ('NFC', {
+        'name': _lazy(u'NFC'),
+        'description': _lazy(u'The app requires access to Near Field '
+                             u'Communication (NFC) API.'),
+        'apis': ('navigator.mozNfc',),
```

Comment:
Yep, I wondered the same thing, had to double check that this wasn't a mistake in the documentation, but no, that's how it's defined, see https://developer.mozilla.org/en-US/docs/Web/API/Navigator/mozNfc and https://developer.mozilla.org/en-US/docs/Web/API/NFC_API/Using_the_NFC_API